### PR TITLE
Added Honduras municipalities and support for a sub-directory in a package with additional svg maps

### PR DESCRIPTION
**Pull request by avalladaresm**
Hello Victor, I'd like to add a new Honduras map that shows its municipalities.

Also, this is probably a new feature that I'd like you to test and tell me what you think. So, I changed the svgToJs.fs file to look for an `extra` directory inside a package directory that may contain additional svg files, for example, in the case of this new component, a file for each department showing the divisions of its municipalities, then an index.js file is created inside the `extra` directory containing objects of all the svg files in that directory, and then its usage is (for example): `import { Atlantida } from @svg-maps/honduras.municipalities/extra`. 
I would really appreciate it for you to support this feature, and to take a look at the changes and provide any feedback. :smiley: 

Thank you!

### CODE REVIEW

**Review by VictorCazanave — COMMENTED** [review]
Thanks a lot again for your contribution @avalladaresm ! 😃 

I understand your interesting new feature, but I think there are 2 issues:
- one goal of a monorepo is to provide independent packages. If someone wants a map (e.g. of a department), he shouldn't have to download other maps (e.g. all the other departments)
- I'm thinking about limiting this repository only to maps of countries, because adding maps of states, departments, cities... may become hard to organize/maintain. Currently [my idea](https://github.com/VictorCazanave/svg-maps/issues/52) is to allow people to publish their own maps (other than country) and add a link in this repository.

Sorry I couldn't support this feature. If you have an other idea, I'm open to discussion!

**Comment by avalladaresm** [comment]
Hey @VictorCazanave,

Thanks for your insight, I'll try to look into fixing what you mentioned on your first point. Secondly, I'll fork your project and have it include each of the departments' maps so you can link it in the main `README.md` of your project, I just have a couple of questions, what do you mean exactly with the TODO mentioned in #52? Where would I need to publish the `svgToJs.js` I modified? I used a desktop program to merge the different administrative boundaries level maps provided by [GADM](www.gadm.org), so would I have to mention the program's name instead linking an online tool? And what about the 2nd TODO, Update `CONTRIBUTING.md`? That's not very clear to me.

Thank you for offering that solution though.

File: packages/honduras.municipalities/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+These data were extracted from the GADM database (www.gadm.org), version 3.4, April 2018. They can be used for non-commercial purposes only.  It is not allowed to redistribute these data, or use them for commercial purposes, without prior consent. See the website (www.gadm.org) for more information.
```
[the remark this answers is not in the document]
Comment (avalladaresm) [reply packages/honduras.municipalities/LICENSE.md:1]: I already sent them a message, I'll let you know what they tell me.

**Comment by VictorCazanave** [comment]
Sorry for the unclear issue https://github.com/VictorCazanave/svg-maps/issues/52, the TODO is actually for myself 😄 
In order to allow people to publish their own maps, **I should**:
- publish `svgToJs.js`, then people can install it (e.g. `npm install @svg-maps/script`) and use it in their repository
- update `CONTRIBUTING.md` to explain when and how to publish your own map

Then people are free to create (copy, merge...), organize (1 monorepo with multiple packages, 1 package with multiple maps...), document (license, usage...) their maps as they want.
Forking this repository is not needed to publish a custom map.

I hope it's a bit more clear 😅

**Comment by avalladaresm** [comment]
Oh gotcha, thanks for explaining. Well, as you may have noticed, at least with how I organized the different department maps, the `svgToJs.js` works, so I will only have to wait for you to complete your TODO and create a pull request with a link to my package :smiley:

**Comment by VictorCazanave** [comment]
@avalladaresm I finally had time to to work on the [CLI tool](https://www.npmjs.com/package/@svg-maps/cli) and the [documentation](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md) 😅 

Feel free to open a PR to add a link to your package!

**Pull request closed without merging**